experimenter/manifesttool/version.py

```diff
@@ -145,6 +167,8 @@ def parse_version_file(f: VersionFile, contents: str) -> Optional[Version]:
     Returns:
         The parsed Version.
     """
+    from manifesttool.appconfig import VersionFileType
```

Comment:
Same here

Reply:
This is also a common pattern for avoiding circular imports

Reply:
Yep, the import doesnt actually happen until the function executes, which lets us have the app config module import this module for the version definitions. Good question :)